src/components/emote-picker-on-input/EmotePickerOnInput.jsx

```diff
@@ -8,7 +8,7 @@ import SearchBar from '../search-bar/SearchBar';
 const EmotePickerOnInput = ({ inputElem }) => {
   const [searchString, setSearchString] = useState('');
 
-  const src = chrome.runtime.getURL('icon-34.png');
+  const src = chrome.runtime.getURL('icon-28.png');
```

Comment:
this file is neither included in this repository nor in this PR ... or am I missing something?
![missing](https://user-images.githubusercontent.com/25775076/215543703-d563cee1-059c-4b7b-8a33-7bc1fb90e9c9.PNG)